.github/workflows/analysis-of-endpoint-connections.yml

```diff
@@ -23,6 +23,24 @@ jobs:
         run: |
           git diff --name-only origin/${{ github.event.pull_request.base.ref }} HEAD > modified_files.txt
 
+      # Analyze the client sided REST-API calls
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20.8.0'
+
+      - name: Install and compile TypeScript
+        run: |
+          cd supporting_scripts/analysis-of-endpoint-connections/src/main/typeScript/
+          npm install
+          npm install typescript
+          tsc -p tsconfig.analysisOfEndpointConnections.json
```

Comment:
the `-p` flag allows me to set the path of the specific `tsconfig` file I want to use to compile my project